sail/Models/EntryLayout.php

```diff
@@ -726,7 +744,7 @@ private function createWithoutPermission(LocaleField $titles, Collection $schema
         $author = User::$currentUser ?? User::anonymousUser();
         $authors = Authors::init($author);
 
-        $slug = !isset($slug) ? Text::slugify($titles->{Locale::default()}) : $slug;
+        $slug = $slug ?? Text::from($titles->{Locale::default()})->slug()->value();
```

Comment:
I think we should keep the !isset($slug) because there is a diffenrence between null and empty... and I'm not sure that with ?? it'll be ok

Reply:
ok

Reply:
phpstorm keeps telling me that ?? is perfect for it.